lib/CArtHandler.cpp

```diff
@@ -752,6 +757,48 @@ void CArtifactInstance::init()
 	setNodeType(ARTIFACT_INSTANCE);
 }
 
+std::string CArtifactInstance::getEffectiveDescription(
+	const CGHeroInstance *hero) const
+{
+	std::string text = this->artType->Description();

```

Comment:
That is again coding style related, but is there some reasoning behind using "this->" prefixes there? As far as I remember in VCMI codebase we currently only use them to avoid conflicts with local variables and few places with some weird compiler issues.

Otherwise everything is okay.